generatorreceiver/internal/topology/service_route_test.go

```diff
@@ -2,10 +2,11 @@ package topology
 
 import (
 	"fmt"
+	"testing"
+
```

Comment:
another tiny nit: alphabetize the imports here and delete the empty line

Reply:
Same as my other follow-up, ignore that comment^ and keep the empty line and the order you have!